content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -151,13 +155,24 @@ result.forEach((x) => {
 
 > **Note**: Prisma sends JavaScript integers to PostgreSQL as `INT8`. This might conflict with your user-defined functions that accept only `INT4` as input. If you use `$queryRaw` in conjunction with a PostgreSQL database, update the input types to `INT8`, or cast your query parameters to `INT4`.
 
+#### Dynamic table names in PostGres
```

Comment:
```suggestion
#### Dynamic table names in Postgres
```

Reply:
Actually, official name we use everywhere (I hope) 😆 :
```suggestion
#### Dynamic table names in PostgreSQL
```

Reply:
Thanks Jan - I just saw this - for some reason I only saw Harshit's suggestion. I've already made this change and have now merged the branch.

Reply:
Great to see we think alike :D